sdk/webpubsub/azure-messaging-webpubsubclient/azure/messaging/webpubsubclient/_client.py

```diff
@@ -683,13 +721,13 @@ def _start_core(self):
         self._url = self._credential.get_client_access_url()
         self._connect(self._url)
 
-    def _start(self) -> None:
+    def start(self) -> None:
```

Comment:
Are we able to make the start implicit rather than have an explicit method for this?
For our HTTP and AMQP clients, usually the client will be automatically started when I being to use it.

Reply:
I rename `start` to `open`. If no `open()`, users can't restart the client after it closes. And .Net/Java/Typescript SDK all have public `start/stop` function.